jsonnet/telemeter/metrics.jsonnet

```diff
@@ -73,4 +73,38 @@
   // cluster:node_instance_type_count:sum is the number of nodes
   // of each instance type and role.
   '{__name__="cluster:node_instance_type_count:sum"}',
+  //
+  //OCS metrics to be collected:
+  // job:ceph_osd_in:count gives the count of ceph osd which are in 'in' state.
+  '{__name__="job:ceph_osd_in:count"}',
```

Comment:
Unfortunately these are all incredibly specific - we need to have higher level summaries here.  This is a substantial portion of our metrics budget, so we can't add these as is.  We probably need to discuss in detail what the limitations are and how you can fit within a budget.

Reply:
Also, I'd like to see what alerts you have configured for this already if you can link that.

https://github.com/openshift/telemeter/pull/194/files is great - it's a single metric that shows us data about usage.  I asked whether there are a few other ones that show failure rates or whether they have alerts.  I would recommend starting by having 3-10 series that have the highest value input.

Reply:
@smarterclayton understanding the budget we have to play with would have been nice ;). All the metrics we've identified are driven from a support angle - if we can't make them fit within the payload I have some ideas of where compromises could be made. However, there are another 6 metrics coming from Noobaa. Will discuss further with @anmolsachan and @shtripat

Reply:
@smarterclayton at this point we have a list of about 11-12 that cover Ceph and Noobaa.